rsts/user/features/fast_registration.rst

```diff
@@ -9,10 +9,67 @@ Fast Registration
 Are you frustrated by having to wait for an image build in order to test out simple code changes to your Flyte workflows? If you're interested in reducing to your iteration cycle to mere seconds, read on below.
 
 Caveats
-#######
+=======
 
 Fast-registration only works when you're testing out code changes. If you need to update your container, say by installing a dependency or modifying a Dockerfile, you **must** use the conventional method of committing your changes and rebuilding a container image.
 
+Prerequisites
+=============
+
+* Upgrade your flytekit dependency to ``>=0.16.0`` and re-run piptools if necessary
```

Comment:
this has to be beta still.

Reply:
but eventually it won't be?

Reply:
Yeah i think in a couple more weeks maybe we'll release everything as 0.16.0

Reply:
@EngHabu